### PR TITLE
Adding r-ravedash and R dependencies

### PR DESCRIPTION
**Pull request by dipterix**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: .gitignore
Comment (mfansler) [inline .gitignore:1]: This should not be edited. Remove all such changes from branch.

File: recipes/r-filearray/meta.yaml
Comment (mfansler) [inline recipes/r-filearray/meta.yaml:66]: Typically, R packages should include `conda-forge/r` as a maintainer.

**Review by mfansler — CHANGES_REQUESTED** [review]

**Comment by dipterix** [comment]
@conda-forge-admin, please rerender

**Comment by dipterix** [comment]
@mfansler My current PR failed because one dependence `r-hdf5r` package wasn't supported on Windows: `Unsatisfiable dependencies for platform win-64: {MatchSpec("r-hdf5r[version='>=1.3.4']")}`. This issue has been fixed with a recent PR (https://github.com/conda-forge/r-hdf5r-feedstock/pull/26). 

Now I would like to trigger re-check. I remember seeing instructions somewhere, but I couldn't remember. Would you mind showing me how to do it? Thanks : )

Also I have revised the PR based on your requests.

**Comment by mfansler** [comment]
Why so many packages together? Does `r-filearray` require all these additional packages to be added?

**Comment by dipterix** [comment]
@mfansler filearray does not depend on any of other packages. I uploaded part of R packages under my name (filearray, ravetools, dipsaus, threeBrain, raveio, ravedash, rpymat, shidashi) and their dependence (7 other packages). 

Please let me know if you want to separate these packages. I was intended to submit them individually, but was instructed to put all together in the same PR (https://github.com/conda-forge/staged-recipes/pull/20778).

**Comment by mfansler** [comment]
@dipterix Apologies if there was a miscommunication. My previous comment was to submit *interdependent* recipes together. For example, if you have a package, **A**, you wish to submit, and it _requires_ packages **B** and **C** that are not yet available, then you can put those together with a submission for **A**.

In the future it would still be preferable to submit independent packages as separate recipes. For now, I'll review it as is.

File: recipes/r-pkgfilecache/meta.yaml
Comment (mfansler) [inline recipes/r-pkgfilecache/meta.yaml:57]: Additional maintainers must comment that they wish to be included.

File: recipes/r-gifti/meta.yaml
Comment (mfansler) [inline recipes/r-gifti/meta.yaml:57]: Additional maintainers must comment that they wish to be included.

File: recipes/r-oro.nifti/meta.yaml
Comment (mfansler) [inline recipes/r-oro.nifti/meta.yaml:60]: Additional maintainers must comment that they wish to be included.

File: recipes/r-ravedash/meta.yaml
Comment (mfansler) [inline recipes/r-ravedash/meta.yaml:73]: Additional maintainers must comment that they wish to be included.

File: recipes/r-raveio/meta.yaml
Comment (mfansler) [inline recipes/r-raveio/meta.yaml:95]: Additional maintainers must comment that they wish to be included.

File: recipes/r-ravetools/meta.yaml
Comment (mfansler) [inline recipes/r-ravetools/meta.yaml:83]: Additional maintainers must comment that they wish to be included.

File: recipes/r-freesurferformats/meta.yaml
Comment (mfansler) [inline recipes/r-freesurferformats/meta.yaml:68]: Additional maintainers must comment that they wish to be included.

**Review by mfansler — CHANGES_REQUESTED** [review]
Recipes look great, but some need users to acknowledge in this thread that they wish to be maintainers, or remove them from the recipe maintainer list. I'd also suggest you rename the title of the thread to something like "Adding r-ravedash and R dependencies", since it does appear `r-ravedash` is the "leaf" package that requires all the other packages.

**Comment by beauchamplab** [comment]
I wish to be included as the maintainer of the packages listed above (`r-ravetools`, `r-raveio`, `r-ravedash`)

File: recipes/r-freesurferformats/meta.yaml
Comment (dfsp-spirit) [reply recipes/r-freesurferformats/meta.yaml:68]: I wish to be included.

File: recipes/r-pkgfilecache/meta.yaml
Comment (dfsp-spirit) [reply recipes/r-pkgfilecache/meta.yaml:57]: I want to be included.

**Review by mfansler — APPROVED** [review]
@conda-forge/staged-recipes this looks ready to merge 🙏

File: recipes/r-gifti/meta.yaml
Comment (cbrueffer) [inline recipes/r-gifti/meta.yaml:45]:
```suggestion
  license: GPL-2.0-only
```

File: recipes/r-rpymat/LICENSE.md
Comment (cbrueffer) [inline recipes/r-rpymat/LICENSE.md:1]: This file can be deleted, the `licence_file` definition in `meta.yaml` below should pick up the file shipped in the tarball.

File: recipes/r-gifti/meta.yaml
Comment (dipterix) [reply recipes/r-gifti/meta.yaml]: Thanks for pointing this out. It has been fixed in the latest commit

File: recipes/r-rpymat/LICENSE.md
Comment (dipterix) [reply recipes/r-rpymat/LICENSE.md]: I see. I have deleted this file in the latest commit

Comment (dipterix) [reply recipes/r-rpymat/LICENSE.md]: @cbrueffer , The [CI check](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=599781&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823) suggested that `rpymat` license file was not found even though the file `LICENSE.md` is in the tarball...

Comment (cbrueffer) [reply recipes/r-rpymat/LICENSE.md]: The issue is that the license is part of the repository, but not packaged as part of the tarball. The quick fix is to re-add the license here (sorry!). I see you're the upstream developer; it would be great if you could package the license file as part of your software.

Comment (dipterix) [reply recipes/r-rpymat/LICENSE.md]: Thanks, this is actually my fault. I forgot to compile the LICENSE file in `rpymat`. Will include the LICENSE file in the next submit. However, that's not gonna happen very soon. I agree the quickest fix is to add back LICENSE.md. Once the recipe is merged into conda-forge and new version is submitted to CRAN, I will create another PR to fix the LICENSE issue in rpymat. 

Waiting for the CI to finish...

Thanks again for point this issue out : )

**Review by cbrueffer — APPROVED** [review]

File: recipes/r-rpymat/LICENSE.md
Comment (cbrueffer) [reply recipes/r-rpymat/LICENSE.md]: No problem! Sounds like a plan.

**Pull request merged**
